core/vm/gas_table.go

```diff
@@ -88,102 +86,14 @@ func memoryCopierGas(stackpos int) gasFunc {
 	}
 }
 
-func gasExtCodeSize(evm *EVM, contract *Contract, stack *Stack, mem *Memory, memorySize uint64) (uint64, error) {
```

Comment:
want to add this back in.  We need to charge out of gas for `extCodeSize`, `extCodeCopy` in `gas_table.go` before looking up the value in the DB (in `instructions.go`).

Reply:
also `gasSload`.

Reply:
also `gasCall`.